catalog/transformer/catalog-transformer-xlsx/src/main/java/ddf/catalog/transformer/xlsx/XlsxMetacardTransformer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.xlsx;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class XlsxMetacardTransformer implements MetacardTransformer {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(XlsxMetacardTransformer.class);
+
+  private XlsxMetacardUtility xlsxMetacardUtility;
+
+  public XlsxMetacardTransformer() {
+    this.xlsxMetacardUtility = new XlsxMetacardUtility();
+  }
+
+  @Override
+  public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
+      throws CatalogTransformerException {
+
+    if (metacard == null) {
+      LOGGER.debug("Attempted to transform null metacard");
+      throw new CatalogTransformerException("Null metacard cannot be transformed to XLSX");
+    }
+
+    return xlsxMetacardUtility.buildSpreadSheet(Collections.singletonList(metacard));
```

Comment:
✏️ Personally I'd like it if these two operations were broken down into two lines for readability. 

```
List<Metacard> metacards = Collections.singletonList(metacard);
return xlsxMetacardUtility.buildSpreadSheet(metacards);
```